scheduler/src/cook/mesos/util.clj

```diff
@@ -229,13 +229,11 @@
 ;; are set at the same time, in "real" time. This means that
 ;; jobs submitted after `start` will have been created after
 ;; expanded start
-(defn get-jobs-by-user-and-states
-  "Returns all job entities for a particular user
-   in a particular state, in the specified timeframe,
-   without a custom executor."
-  [db user states start end limit]
-  (let [states (set states)
-        ;; Expand the time range so that clock skew between cook
+(defn get-completed-jobs-by-user
+  "Returns all completed job entities for a particular user
+   in the specified timeframe, without a custom executor."
```

Comment:
Oh shit, we aren't filtering out custom executor jobs. Can you add another filter for that?